src/flake8_requirements/modules.py

```diff
@@ -657,11 +657,13 @@
     "pillow": ["PIL"],
     "pillow-simd": ["PIL"],
     "pip-tools": ["piptools"],
+    "plotly": ["plotly", "_plotly_utils"],
```

Comment:
I can see that plotly exports more packages: https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/setup.py#L521 All exported packages should be listed here (for brevity's sake `plotly.*` can be added as a single `plotly`).

Reply:
Good catch. Thanks.